docs/learn/learn-staking-miner.md

```diff
@@ -48,7 +51,28 @@ order to submit solutions to the NPoS election. When the correct time comes, it
 solution and submits it to the chain. The default miner algorithm is sequential Phragmén with a
 configurable number of balancing iterations that improve the score.
 
-![NPoS election optimization](../assets/staking-miner/NPoS-election-optimization.png)
+## Deposit and reward mechanics
+
+The staking miners are required to pay a deposit in order to post their solutions. The deposit is a
+sum of the `SignedDepositBase`, which is a fixed amount and `SignedDepositByte`, a variable amount
+per KB of solution data. All good solutions are subject to recieving a `SignedRewardBase`.
+
+Current deposit(**SignedDepositBase**) is
+{{ polkadot: <RPC network="polkadot" path="consts.electionProviderMultiPhase.signedDepositBase" defaultValue={16} filter="humanReadable"/> :polkadot }}
+{{ kusama: <RPC network="kusama" path="consts.electionProviderMultiPhase.signedDepositBase" defaultValue={16} filter="humanReadable"/> :kusama }}
+
+Current deposit per byte(**SignedDepositByte**) is
+{{ polkadot: <RPC network="polkadot" path="consts.electionProviderMultiPhase.signedDepositByte" defaultValue={16} filter="humanReadable"/> :polkadot }}
+{{ kusama: <RPC network="kusama" path="consts.electionProviderMultiPhase.signedDepositByte" defaultValue={16} filter="humanReadable"/> :kusama }}
+
+Current rewards(**SignedRewardBase**) is
+{{ polkadot: <RPC network="polkadot" path="consts.electionProviderMultiPhase.signedRewardBase" defaultValue={16} filter="humanReadable"/> :polkadot }}
+{{ kusama: <RPC network="kusama" path="consts.electionProviderMultiPhase.signedRewardBase" defaultValue={16} filter="humanReadable"/> :kusama }}
+
+Once the staking miner with a good solution is ready to be rewarded,
+[`finalize_signed_phase_accept_solution`](https://github.com/paritytech/substrate/blob/master/frame/election-provider-multi-phase/src/signed.rs#L453-L474)
```

Comment:
This gives the false impression that some account calls `finalize_signed_phase_accept_solution`. I think it is good that you are trying to back statements with code, but it need to be accurate. 

I would try and paraphrase this: https://github.com/paritytech/substrate/blob/3bbdba405d42d7023eeb297efb03a3ec4ccf9563/frame/election-provider-multi-phase/src/lib.rs#L69-L74

Reply:
Yeah, this is all mentioned below in the signed phase section. Actually makes more sense to move the deposit and reward mechanics below that and it will flow better.